gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/close.ts

```diff
@@ -0,0 +1,45 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+
+export default class SetBilling extends SolanaCommand {
```

Comment:
I know this is still a draft, but we should have 2 distinct close commands iicr, right?

Reply:
Yeah, we should probably go with something like the ownership transfer functionality (https://github.com/smartcontractkit/chainlink-solana/pull/106/files#diff-f84525b723c248009cd2ad7e443e07d6d0cd4d6245197d87a0d77cf9de3d9289R8)

Reply:
Yeah, that looks nice!